components/accept-financial-contributions/HostCollectiveCard.js

```diff
@@ -177,22 +186,22 @@ const HostCollectiveCard = ({ host, collective, onChange, ...props }) => {
         </ModalHeader>
         <ModalBody>
           <Fragment>
-            <HTMLContent content={host.longDescription} />
+            <HTMLContent content={host.description} />
```

Comment:
**[:no_entry_sign: **Blocker**]** Host description is not a rich test and it's not sanitized so we don't need to use `HTMLContent`. I've put this as a blocker because it can be used to inject malicious code, try to set this as the host description:

```es6
<script>alert('Hello');</script>
```

---

<details>
<summary>More details about this feedback</summary>
<br/>

This feedback categorization is inspired by [Netlify's Feedback Ladders system](https://www.netlify.com/blog/2020/03/05/feedback-ladders-how-we-encode-code-reviews-at-netlify). It includes 5 levels of feedback:

:rotating_light: **Critical blocker**
This feedback blocks all related work, and requires immediate action. 

:no_entry_sign: **Blocker**
This feedback should block the work from moving forward or being approved, but it does not necessarily require immediate action from the team.

:hammer_and_wrench: **Important**
This feedback should not block the work from moving forward or being approved, but it does require future action from the team. This generally means the feedback is not required for the MVP or initial draft of this work, but should be addressed at some point and we should ideally create an issue to track it.

:sparkles: **Enhancement**
This feedback should not block the work from moving forward or being approved. The project lead or owner should consider the feedback and implement or open an issue it if they agree with it.

:ice_cream: **Details & personal preferences**
This feedback should not block the work from moving forward or being approved. The project lead or owner should consider the feedback and implement or open an issue it if they deem it useful. This is also the category for everything that relates to personal preferences or code style.

</details>

Reply:
Hm weird, it's definitely `host.description` on my current branch. I must not have pushed that change last night. But thanks for the info on why it's better.